src/services/user/services/AdminUsers.js

```diff
@@ -71,9 +75,12 @@ class AdminUsers {
 			const currentUserId = params.account.userId.toString();
 
 			// fetch base data
-			const [currentUser, roles] = await Promise.all([getCurrentUser(currentUserId), getRoles()]);
+			const [currentUser, roles] = await Promise.all([getCurrentUserInfo(currentUserId), getRoles()]);
 			const { schoolId } = currentUser;
 
+			const currentSchool = await this.app.service('schools').get(schoolId);
+			const currentYear = currentSchool.years.activeYear._id;
```

Comment:
I think `activeYear` can be `null`. I would suggest using `defaultYear`, which is `activeYear || nextYear` and never `null`. @tofixx: please back me up 😄

Reply:
thats completely right @Parnswir @Metauriel, see https://github.com/schul-cloud/schulcloud-server/blob/develop/src/services/school/logic/year.js for further details

Reply:
will do tomorrow